transfer_tests.jl

```diff
@@ -1,16 +1,16 @@
 using IntervalArithmetic
 using Base.Filesystem
+using Glob
 
 output_file_path = joinpath(dirname(dirname(pathof(IntervalArithmetic))), "test", "ITF1788_tests")
 input_file_path = joinpath(@__DIR__, "output", "julia", "Base.Test", "ValidatedNumerics")
 files = readdir(relpath(input_file_path, @__DIR__))
+files = glob("*.jl", input_file_path)
 
 for file in files
-    if(file=="Makefile")
-        continue
-    end
+    bname = basename(file)
     filepath = joinpath(input_file_path, file)
-    ofilepath = joinpath(output_file_path, file)
+    ofilepath = joinpath(output_file_path, bname)
     cp(relpath(filepath, @__DIR__), ofilepath; force=true)
```

Comment:
Do you really need `@__DIR__` here?  
(You might, it just feels wrong.)

Reply:
We don't, I was just generalizing, changes have been made to remove it.